src/frontends/tensorflow/src/helper_ops/undefined_constant.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "internal_operation.hpp"
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+
+class UndefinedConstant : public InternalOperation {
+public:
+    OPENVINO_OP("UndefinedConstant", "ov::frontend::tensorflow::util", InternalOperation);
+
+    UndefinedConstant(const std::shared_ptr<DecoderBase>& decoder = std::make_shared<DecoderFake>())
+        : InternalOperation(decoder, {}, 1) {
+        validate_and_infer_types();
+    }
+
+    void validate_and_infer_types() override {
+        set_output_type(0, ov::element::undefined, ov::PartialShape::dynamic());
```

Comment:
Why type undefined? Not dynamic?

Reply:
if this is string, maybe undefined is okay. I am just concerned that some type arithmetic wont work on it as expected, e.g. merging types, like f32 mege with dynamic will make output f32, but with undefined I am not sure what will happen.

Reply:
This is only used for constants which type we currently not supported. `dynamic` means different thing and is used for dynamic size types. This case is more undefined because we don't know anything about its type.